Automattic/newspack-newsletters
# Update various packages, including colors to use latest WP base-styles

### All Submissions:






### Changes proposed in this Pull Request:

The main change with this PR is the use of the latest `@wordpress/base-styles` package which means that the color variables needed to be updated.
I've also:
* Removed Material packages
* Updated calypso-build to 6.3.0 to match other plugins
* Small bug fixes

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check the various screens

### Other information:
